klaviyo/campaigns.py

```diff
@@ -26,17 +28,19 @@ def get_campaigns(self, page=None, count=None):
 
         return self._v1_request(self.CAMPAIGNS, self.HTTP_GET, params)
 
-    def get_campaign_recipients(self, campaign_id, count=None, sort="asc"):
-        """
-        Returns summary information about email recipients for the campaign
+    def get_campaign_recipients(self, campaign_id, count=25000, sort="asc"):
```

Comment:
can we have the count be the default which is 5000

Reply:
@remstone7 done.